xarray/core/ops.py

```diff
@@ -22,6 +23,13 @@
 
 try:
     import bottleneck as bn
+    if LooseVersion(bn.__version__) < LooseVersion('1.1'):
+        warnings.warn(
```

Comment:
Can we trigger this warning lazily (e.g., when calling actually resample)? If we do this at import time we are going to get a lot of complaints from users using bottleneck < 1.1 about irrelevant warnings.